mathics/builtin/natlang.py

```diff
@@ -119,6 +119,9 @@ def generator(doc):
         'Adjective': [nltk.corpus.wordnet.ADJ, nltk.corpus.wordnet.ADJ_SAT],
         'Adverb': [nltk.corpus.wordnet.ADV],
     }
+except LookupError:
+    _wordnet_pos_to_type = {}
+    _wordnet_type_to_pos = {}
```

Comment:
We should probably print a warning here so the user can fix the issue by installing the missing corpus.